src/components/card/card.less

```diff
@@ -11,54 +11,62 @@
   box-shadow: 0px 1px 1px 0px rgba(0, 0, 0, 0.3);
 
   .avatar{
-    margin-right: 15px;
+    margin-right: 10px;
     img{
       display: block;
     }
   }
 
   .thumbnail {
-    padding: 6px 0 0 0;
+    padding: 0;
     height: 0;
-    padding-bottom: 61.875%;
+    padding-bottom: 64.875%;
```

Comment:
Seems like an odd hack added for image loading trouble here: https://github.com/mozilla/webmaker-core/commit/c44b55cef33a3db2e50f4821c8dcad26b397c354

Did you intend to change it by 3%? Was that to increase space below thumb and above icon and title?